frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/MenuLayout.java

```diff
@@ -136,7 +146,17 @@ public void buttonClick(Button.ClickEvent event) {
                 authService.logout(RequestHolder.getRequest());
                 authCtx.clear();
                 refreshUserBar();
-                navigator.navigateTo(Login.class);
+                String logout_url = null;
```

Comment:
There should be a comment that the lines 149-159 are there only because of SSO LOGOUT_URL.